config.json

```
@@ -44,6 +44,14 @@
         {
             "repo": "https://github.com/iotaledger/identity.rs",
             "staticPath": "./documentation/static"
+        },
+        {
+            "repo": "https://github.com/iotaledger/identity.rs",
+            "staticPath": "./documentation/static"
+        },
+        {
+            "repo": "https://github.com/iotaledger/streams.git",
```

```suggestion
            "repo": "https://github.com/iotaledger/streams",
```

I guess we should be consistent here and don't use `.git`